proto/public.proto

```diff
@@ -383,6 +384,7 @@ message RouteTable {
   message Row {
     string route = 1;
     string time_window = 2;
+    string dst = 6;
```

Comment:
`dst` feels pretty vague to me, especially given that we use dst elsewhere to mean other things. This field holds authorities, as far as I can tell. How would you feel about calling it `authority`?

Reply:
Or... related to the other protobuf change that I suggested, this would be better as:

```
  message Row {
    string route = 1;
    string time_window = 2;
    Resource resource = 6;

    BasicStats stats = 5;
  }
```

Reply:
All we get from Prometheus is the dst label, which the proxy sets to the authority.  So I could see calling it authority instead of dst, possibly.  But we'd have to do a bunch of reverse engineering to construct a Resource from that string.  i.e. create a Service resource if the dst matches the Kubernetes service DNS pattern, otherwise create an Authority resource.